protocol/protobuf/chat_message.proto

```diff
@@ -54,6 +54,22 @@ message DeleteMessage {
   MessageType message_type = 5;
 }
 
+message DiscordMessage {
+  string id = 1;
+  string type = 2;
+  string timestamp = 3;
+  string timestamp_edited = 4;
+  string content = 5;
+  DiscordMessageAuthor author = 6;
+}
+
+message DiscordMessageAuthor {
+  string id = 1;
+  string name = 2;
+  string discriminator = 3;
+  string nickname = 4;
+  string avatar_url = 5;
+}
```

Comment:
When are these protobufs used for protobuf related things?

Reply:
Already answered offline, but for transparency's sake I'll echo my response here:

As part of this PR they indeed only serve as structs. However, later on, when a new `ChatMessage` type for `DiscordMessage` is introduce, this data **has** to live in a protobuf because that's the type we rely on to save messages in the database.

I started out in fact with just simple structs but then had to move them in protobuf to be able to store data.